pkg/slack/app.go

```diff
@@ -225,11 +225,20 @@ func (a *App) messageLoop(ctx context.Context, client *socketmode.Client) {
 
 				switch subcommand {
 				case "subscribe":
+					filterLayers := array.Unique(args[2:])
+					filter, err := NewFilter(filterLayers)
+					if err != nil {
+						a.logger.Warn("failed to subscribe", zap.Error(err))
+						client.Ack(*e.Request, map[string]interface{}{
+							"text": fmt.Sprintf("Failed to subscribe '%s': %s\n", repo, err),
+						})
+					}
```

Comment:
Should early return here. `continue` here

Reply:
Code is already overwritten...